data-explorer/kusto/management/clear-external-artifacts-cache-command.md

```diff
@@ -0,0 +1,52 @@
+---
+title: .clear cluster cache external-artifacts command
+description: Learn how to use the `.clear cluster cache external-artifacts` command to clear cached external-artifacts of language plugins.
+ms.reviewer: 
+ms.topic: reference
+ms.date: 
+---
+# .clear cluster cache external-artifacts command
+
+Clears cached external-artifacts of language plugins.  
+This command can be typically used in cases where you update your external-artifact files in your external storage, while the cache still holds the old ones. In such a case this command clears the cache entries and ensures your next queries run with the updated artifacts.  
+learn more about [language extensions on your cluster](../language-extensions.md).
```

Comment:
```suggestion
```suggestion

The command is useful when you update external-artifact files stored in external storage, while the cache retains the previous versions. In such scenarios, executing this command will clear the cache entries and ensure that subsequent queries run with the latest version of the artifacts.
```